src/libNode/FinalBlockProcessing.cpp

```diff
@@ -782,6 +783,37 @@ bool Node::IsMyShardsMicroBlockStateDeltaHashInFinalBlock(
                isEveryMicroBlockAvailable);
 }
 
+bool Node::IsMyShardsMicroBlockInFinalBlock(const uint256_t& blocknum)
+{
+    if (m_microblock == nullptr)
```

Comment:
Should not you return true here ??

Reply:
Why return true here? if no m_microblock ofcourse my shards not in finalblock

Reply:
if no microblock, that means I did not submit a microblock. Just make sure the implementation works in case of no micro block consensus. Otherwise it is ok.

Reply:
Yup, the purpose is just to make sure there is no microblock in finalblock is related to this node